### PR TITLE
[Bug fix] update pkg requirements

### PR DESCRIPTION
**Pull request by kaixi-wang**
## What changes are proposed in this pull request?
- update pymongo from 3.11 to 4.3.2
  - v3.12 added support for dot notation in field names
  - v4.2 adds support for MongoDB 6.0
  - for ref: https://pymongo.readthedocs.io/en/stable/changelog.html

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Review by benjaminpkane — APPROVED** [review]
LGTM

**Review by brimoor — CHANGES_REQUESTED** [review]
Pause for a sec

File: requirements/common.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ pandas>=1.3
 plotly==5.9.0
 pprintpp==0.4.0
 psutil>=5.7.0
-pymongo>=3.11
+pymongo>=4.3.2
```
Comment (brimoor) [inline requirements/common.txt:21]: Best to update `motor>=3.1.1` too: https://motor.readthedocs.io/en/stable/requirements.html

Comment (kaixi-wang) [reply requirements/common.txt:21]: whoops I thought it was

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**